SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -72,12 +80,16 @@ Cloud Hypervisor is an open source Virtual Machine Monitor (VMM) that runs on to
 
 %prep
 
-%autosetup -p1 -n cloud-hypervisor-msft-v%{version}
+%setup -n cloud-hypervisor-msft-v%{version}
+%patch 1 -p1
 %if 0%{?using_vendored_crates}
 tar xf %{SOURCE1}
 mkdir -p .cargo
 cp %{SOURCE2} .cargo/
 %endif
+# The vendored archive has been populated based on the patch, so we need to
+# repatch here as well in order to use the same versions
+%patch 0 -p0
```

Comment:
The `-p1` and `-p0` mix is going to make applying more patches cumbersome. The most common kind of patches use `-p1`, so could you try updating patch 0 to work with `-p1`? I'll add a comment in the patch itself to show, what I have in mind.
```suggestion
%autosetup -n cloud-hypervisor-msft-v%{version}
%if 0%{?using_vendored_crates}
tar xf %{SOURCE1}
mkdir -p .cargo
cp %{SOURCE2} .cargo/
%endif
# The vendored archive has been populated based on the patch, so we need to
# repatch here as well in order to use the same versions
%autopatch -p1
```